sdk/python/touca/cli/common.py

```diff
@@ -21,6 +21,7 @@ def config_set(options: Dict[str, str]):
     if not config.has_section("settings"):
         config.add_section("settings")
     for key, value in options.items():
-        config.set("settings", key, value)
-        with open(path, "wt") as file:
-            config.write(file)
+        if value is not None:
+            config.set("settings", key, value)
+    with open(path, "wt") as file:
+        config.write(file)
```

Comment:
Unrelated improvement: Fixed unintentional extra indent here that caused multiple writes into the profile.